spec/views/catalog/_index_list_default.html.erb_spec.rb

```diff
@@ -35,5 +37,9 @@
     expect(rendered).to include 'Test depositor_tesim'
     expect(rendered).to include '<span class="attribute-label h4">Rights:</span>'
     expect(rendered).to include 'Test rights_tesim'
+    expect(rendered).to include '<span class="attribute-label h4">Embargo release date:</span>'
+    expect(rendered).to include 'Test embargo_release_date_dtsi'
```

Comment:
Could use some suggestions on the test for the date format. I'm thinking a view test is not ideal, but adding something for the sufia_helper_spec. Thoughts?